### PR TITLE
r/aws_ec2_managed_prefix_list - Fix: `MaxEntries` and `Entry` can be changed in same apply

### LINKED ISSUE #26639: Prefix List ressource won't allow adding an entry and changing the maximum number of entries at the same time
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

windows_amd64
terraform v1.2.7

terraform providers
Providers:
├── provider[[registry.terraform.io/hashicorp/aws](http://registry.terraform.io/hashicorp/aws)] 4.29.0

### Affected Resource(s)

* aws_ec2_managed_prefix_list

### Terraform Configuration Files

Please include all Terraform configurations required to reproduce the bug. Bug reports without a functional reproduction may be closed without investigation.

```hcl
resource "aws_ec2_managed_prefix_list" "my_prefix_list" {
  name           = "my-prefix-list"
  address_family = "IPv4"
  max_entries    = 1

  entry {
    cidr        = "172.217.0.0/32"
    description = "test-block-1"
  }
}
```

### Expected Behavior

The execution of terraform apply + confirmation should have worked.

### Actual Behavior

Received an error: error updating EC2 Managed Prefix List (pl-0625f89a976b5cd): InvalidParameterCombination: You cannot modify the entries and the maximum entries for the prefix list in the same request.

### Steps to Reproduce

1. `terraform apply`, confirm
2. change the terraform configuration to 
```hcl
resource "aws_ec2_managed_prefix_list" "my_prefix_list" {
  name           = "my-prefix-list"
  address_family = "IPv4"
  max_entries    = 2 // increased by one

  entry {
    cidr        = "172.217.0.0/32"
    description = "test-block-1"
  }

  // added this entry
  entry {
    cidr        = "216.58.190.0/32"
    description = "test-block-2"
  }
}
```
3.  `terraform apply`, confirm

### PR DESCRIPTION
**Pull request by drewmullen**
Closes #26639

Output from acceptance testing:

```
$ make testacc TESTS=TestAccVPCManagedPrefixList PKG=ec2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/ec2/... -v -count 1 -parallel 20 -run='TestAccVPCManagedPrefixList'  -timeout 180m
=== RUN   TestAccVPCManagedPrefixListDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListDataSource_basic
=== RUN   TestAccVPCManagedPrefixListDataSource_filter
=== PAUSE TestAccVPCManagedPrefixListDataSource_filter
=== RUN   TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== PAUSE TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== RUN   TestAccVPCManagedPrefixListEntry_ipv6
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv6
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccVPCManagedPrefixListEntry_description
=== PAUSE TestAccVPCManagedPrefixListEntry_description
=== RUN   TestAccVPCManagedPrefixListEntry_disappears
=== PAUSE TestAccVPCManagedPrefixListEntry_disappears
=== RUN   TestAccVPCManagedPrefixList_basic
=== PAUSE TestAccVPCManagedPrefixList_basic
=== RUN   TestAccVPCManagedPrefixList_disappears
=== PAUSE TestAccVPCManagedPrefixList_disappears
=== RUN   TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== PAUSE TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== RUN   TestAccVPCManagedPrefixList_Entry_cidr
=== PAUSE TestAccVPCManagedPrefixList_Entry_cidr
=== RUN   TestAccVPCManagedPrefixList_Entry_description
=== PAUSE TestAccVPCManagedPrefixList_Entry_description
=== RUN   TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== PAUSE TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== RUN   TestAccVPCManagedPrefixList_name
=== PAUSE TestAccVPCManagedPrefixList_name
=== RUN   TestAccVPCManagedPrefixList_tags
=== PAUSE TestAccVPCManagedPrefixList_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListsDataSource_basic
=== RUN   TestAccVPCManagedPrefixListsDataSource_tags
=== PAUSE TestAccVPCManagedPrefixListsDataSource_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_noMatches
=== PAUSE TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixListDataSource_basic
=== CONT  TestAccVPCManagedPrefixList_disappears
=== CONT  TestAccVPCManagedPrefixList_name
=== CONT  TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== CONT  TestAccVPCManagedPrefixListsDataSource_tags
=== CONT  TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== CONT  TestAccVPCManagedPrefixListsDataSource_basic
=== CONT  TestAccVPCManagedPrefixList_Entry_description
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== CONT  TestAccVPCManagedPrefixList_Entry_cidr
=== CONT  TestAccVPCManagedPrefixList_basic
=== CONT  TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== CONT  TestAccVPCManagedPrefixListEntry_description
=== CONT  TestAccVPCManagedPrefixListEntry_disappears
=== CONT  TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4
=== CONT  TestAccVPCManagedPrefixListEntry_ipv6
=== CONT  TestAccVPCManagedPrefixList_tags
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== CONT  TestAccVPCManagedPrefixListDataSource_filter
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidTypeError (11.44s)
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
--- PASS: TestAccVPCManagedPrefixListDataSource_matchesTooMany (21.29s)
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidCIDR (17.40s)
--- PASS: TestAccVPCManagedPrefixList_disappears (81.41s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_noMatches (82.23s)
--- PASS: TestAccVPCManagedPrefixListDataSource_basic (83.32s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_basic (84.19s)
--- PASS: TestAccVPCManagedPrefixListDataSource_filter (87.24s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_tags (95.10s)
--- PASS: TestAccVPCManagedPrefixListEntry_disappears (96.67s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4Multiple (101.23s)
--- PASS: TestAccVPCManagedPrefixList_AddressFamily_ipv6 (107.37s)
--- PASS: TestAccVPCManagedPrefixListEntry_description (107.97s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv6 (108.65s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4 (108.81s)
--- PASS: TestAccVPCManagedPrefixList_name (131.27s)
--- PASS: TestAccVPCManagedPrefixList_Entry_description (134.53s)
--- PASS: TestAccVPCManagedPrefixList_basic (139.63s)
--- PASS: TestAccVPCManagedPrefixList_Entry_cidr (141.11s)
--- PASS: TestAccVPCManagedPrefixList_tags (151.47s)
--- PASS: TestAccVPCManagedPrefixList_updateEntryAndMaxEntry (158.78s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ec2	162.623
...
```

### CODE REVIEW

**Comment by drewmullen** [comment]
```
$ make testacc TESTS=TestAccVPCManagedPrefixList PKG=ec2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/ec2/... -v -count 1 -parallel 20 -run='TestAccVPCManagedPrefixList'  -timeout 180m
=== RUN   TestAccVPCManagedPrefixListDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListDataSource_basic
=== RUN   TestAccVPCManagedPrefixListDataSource_filter
=== PAUSE TestAccVPCManagedPrefixListDataSource_filter
=== RUN   TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== PAUSE TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== RUN   TestAccVPCManagedPrefixListEntry_ipv6
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv6
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccVPCManagedPrefixListEntry_description
=== PAUSE TestAccVPCManagedPrefixListEntry_description
=== RUN   TestAccVPCManagedPrefixListEntry_disappears
=== PAUSE TestAccVPCManagedPrefixListEntry_disappears
=== RUN   TestAccVPCManagedPrefixList_basic
=== PAUSE TestAccVPCManagedPrefixList_basic
=== RUN   TestAccVPCManagedPrefixList_disappears
=== PAUSE TestAccVPCManagedPrefixList_disappears
=== RUN   TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== PAUSE TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== RUN   TestAccVPCManagedPrefixList_Entry_cidr
=== PAUSE TestAccVPCManagedPrefixList_Entry_cidr
=== RUN   TestAccVPCManagedPrefixList_Entry_description
=== PAUSE TestAccVPCManagedPrefixList_Entry_description
=== RUN   TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== PAUSE TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== RUN   TestAccVPCManagedPrefixList_name
=== PAUSE TestAccVPCManagedPrefixList_name
=== RUN   TestAccVPCManagedPrefixList_tags
=== PAUSE TestAccVPCManagedPrefixList_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListsDataSource_basic
=== RUN   TestAccVPCManagedPrefixListsDataSource_tags
=== PAUSE TestAccVPCManagedPrefixListsDataSource_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_noMatches
=== PAUSE TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixList_disappears
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4
=== CONT  TestAccVPCManagedPrefixList_Entry_description
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== CONT  TestAccVPCManagedPrefixList_basic
=== CONT  TestAccVPCManagedPrefixListEntry_disappears
=== CONT  TestAccVPCManagedPrefixListDataSource_basic
=== CONT  TestAccVPCManagedPrefixListEntry_description
=== CONT  TestAccVPCManagedPrefixListEntry_ipv6
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== CONT  TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== CONT  TestAccVPCManagedPrefixListDataSource_filter
=== CONT  TestAccVPCManagedPrefixList_name
=== CONT  TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixListsDataSource_tags
=== CONT  TestAccVPCManagedPrefixListsDataSource_basic
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== CONT  TestAccVPCManagedPrefixList_tags
=== CONT  TestAccVPCManagedPrefixList_Entry_cidr
=== CONT  TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidTypeError (19.99s)
=== CONT  TestAccVPCManagedPrefixList_AddressFamily_ipv6
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidCIDR (23.31s)
--- PASS: TestAccVPCManagedPrefixListDataSource_matchesTooMany (23.60s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_noMatches (90.26s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_basic (92.03s)
--- PASS: TestAccVPCManagedPrefixListDataSource_filter (92.30s)
--- PASS: TestAccVPCManagedPrefixListDataSource_basic (94.19s)
--- PASS: TestAccVPCManagedPrefixList_disappears (95.38s)
--- PASS: TestAccVPCManagedPrefixListEntry_disappears (108.47s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_tags (108.96s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4Multiple (114.18s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv6 (120.69s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4 (122.56s)
--- PASS: TestAccVPCManagedPrefixListEntry_description (123.29s)
--- PASS: TestAccVPCManagedPrefixList_AddressFamily_ipv6 (106.84s)
--- PASS: TestAccVPCManagedPrefixList_name (151.43s)
--- PASS: TestAccVPCManagedPrefixList_Entry_description (153.09s)
--- PASS: TestAccVPCManagedPrefixList_basic (158.64s)
--- PASS: TestAccVPCManagedPrefixList_Entry_cidr (160.80s)
--- PASS: TestAccVPCManagedPrefixList_tags (173.66s)
--- PASS: TestAccVPCManagedPrefixList_updateEntryAndMaxEntry (180.39s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/ec2	184.836s
```

File: internal/service/ec2/vpc_managed_prefix_list.go [hunk 1]
```diff
@@ -182,9 +183,31 @@ func resourceManagedPrefixListRead(ctx context.Context, d *schema.ResourceData,
 func resourceManagedPrefixListUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	conn := meta.(*conns.AWSClient).EC2Conn
 
-	if d.HasChangesExcept("tags", "tags_all") {
+	id := d.Id()
```
Comment (AdamTylerLynch) [inline internal/service/ec2/vpc_managed_prefix_list.go:186]: I am assuming the reason for pulling this into a local variable was to reduce memory?

**Review by AdamTylerLynch — APPROVED** [review]
LGTM (just one question on the local variable).

File: internal/service/ec2/vpc_managed_prefix_list.go (hunk 1 above)
Comment (drewmullen) [reply internal/service/ec2/vpc_managed_prefix_list.go:186]: Yeah, I figured why call a function 10x times instead just ref a var

**Comment by drewmullen** [comment]
```
make testacc TESTS=TestAccVPCManagedPrefixList PKG=ec2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/ec2/... -v -count 1 -parallel 20 -run='TestAccVPCManagedPrefixList'  -timeout 180m
=== RUN   TestAccVPCManagedPrefixListDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListDataSource_basic
=== RUN   TestAccVPCManagedPrefixListDataSource_filter
=== PAUSE TestAccVPCManagedPrefixListDataSource_filter
=== RUN   TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== PAUSE TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4
=== RUN   TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== RUN   TestAccVPCManagedPrefixListEntry_ipv6
=== PAUSE TestAccVPCManagedPrefixListEntry_ipv6
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== RUN   TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== PAUSE TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== RUN   TestAccVPCManagedPrefixListEntry_description
=== PAUSE TestAccVPCManagedPrefixListEntry_description
=== RUN   TestAccVPCManagedPrefixListEntry_disappears
=== PAUSE TestAccVPCManagedPrefixListEntry_disappears
=== RUN   TestAccVPCManagedPrefixList_basic
=== PAUSE TestAccVPCManagedPrefixList_basic
=== RUN   TestAccVPCManagedPrefixList_disappears
=== PAUSE TestAccVPCManagedPrefixList_disappears
=== RUN   TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== PAUSE TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== RUN   TestAccVPCManagedPrefixList_Entry_cidr
=== PAUSE TestAccVPCManagedPrefixList_Entry_cidr
=== RUN   TestAccVPCManagedPrefixList_Entry_description
=== PAUSE TestAccVPCManagedPrefixList_Entry_description
=== RUN   TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== PAUSE TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== RUN   TestAccVPCManagedPrefixList_name
=== PAUSE TestAccVPCManagedPrefixList_name
=== RUN   TestAccVPCManagedPrefixList_tags
=== PAUSE TestAccVPCManagedPrefixList_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_basic
=== PAUSE TestAccVPCManagedPrefixListsDataSource_basic
=== RUN   TestAccVPCManagedPrefixListsDataSource_tags
=== PAUSE TestAccVPCManagedPrefixListsDataSource_tags
=== RUN   TestAccVPCManagedPrefixListsDataSource_noMatches
=== PAUSE TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixListDataSource_basic
=== CONT  TestAccVPCManagedPrefixList_disappears
=== CONT  TestAccVPCManagedPrefixList_name
=== CONT  TestAccVPCManagedPrefixList_Entry_description
=== CONT  TestAccVPCManagedPrefixList_Entry_cidr
=== CONT  TestAccVPCManagedPrefixList_AddressFamily_ipv6
=== CONT  TestAccVPCManagedPrefixListsDataSource_noMatches
=== CONT  TestAccVPCManagedPrefixList_updateEntryAndMaxEntry
=== CONT  TestAccVPCManagedPrefixList_tags
=== CONT  TestAccVPCManagedPrefixListsDataSource_tags
=== CONT  TestAccVPCManagedPrefixListsDataSource_basic
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidCIDR
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4
=== CONT  TestAccVPCManagedPrefixListEntry_ipv6
=== CONT  TestAccVPCManagedPrefixListEntry_expectInvalidTypeError
=== CONT  TestAccVPCManagedPrefixListEntry_ipv4Multiple
=== CONT  TestAccVPCManagedPrefixListDataSource_matchesTooMany
=== CONT  TestAccVPCManagedPrefixList_basic
=== CONT  TestAccVPCManagedPrefixListEntry_disappears
=== CONT  TestAccVPCManagedPrefixListEntry_description
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidTypeError (19.02s)
=== CONT  TestAccVPCManagedPrefixListDataSource_filter
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidCIDR (24.78s)
--- PASS: TestAccVPCManagedPrefixListDataSource_matchesTooMany (27.63s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_basic (109.67s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_noMatches (110.44s)
--- PASS: TestAccVPCManagedPrefixList_disappears (110.81s)
--- PASS: TestAccVPCManagedPrefixListDataSource_basic (115.42s)
--- PASS: TestAccVPCManagedPrefixListDataSource_filter (104.32s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_tags (127.56s)
--- PASS: TestAccVPCManagedPrefixListEntry_disappears (130.04s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4Multiple (138.98s)
--- PASS: TestAccVPCManagedPrefixListEntry_description (145.15s)
--- PASS: TestAccVPCManagedPrefixList_AddressFamily_ipv6 (145.34s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4 (145.80s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv6 (146.82s)
--- PASS: TestAccVPCManagedPrefixList_name (171.59s)
--- PASS: TestAccVPCManagedPrefixList_Entry_description (172.50s)
--- PASS: TestAccVPCManagedPrefixList_basic (177.51s)
--- PASS: TestAccVPCManagedPrefixList_Entry_cidr (178.49s)
--- PASS: TestAccVPCManagedPrefixList_tags (193.17s)
--- PASS: TestAccVPCManagedPrefixList_updateEntryAndMaxEntry (198.68s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/ec2        202.780s
```

**Review by gdavison — APPROVED** [review]
Looks good 🚀 

--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidTypeError (28.24s)
--- PASS: TestAccVPCManagedPrefixListEntry_expectInvalidCIDR (30.86s)
--- PASS: TestAccVPCManagedPrefixListDataSource_matchesTooMany (32.44s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_basic (125.55s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_noMatches (127.06s)
--- PASS: TestAccVPCManagedPrefixList_disappears (134.19s)
--- PASS: TestAccVPCManagedPrefixListDataSource_basic (139.58s)
--- PASS: TestAccVPCManagedPrefixListDataSource_filter (125.30s)
--- PASS: TestAccVPCManagedPrefixListEntry_disappears (158.56s)
--- PASS: TestAccVPCManagedPrefixListsDataSource_tags (158.59s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4Multiple (163.16s)
--- PASS: TestAccVPCManagedPrefixList_AddressFamily_ipv6 (169.57s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv4 (170.54s)
--- PASS: TestAccVPCManagedPrefixListEntry_ipv6 (172.96s)
--- PASS: TestAccVPCManagedPrefixListEntry_description (173.32s)
--- PASS: TestAccVPCManagedPrefixList_name (206.62s)
--- PASS: TestAccVPCManagedPrefixList_Entry_description (211.00s)
--- PASS: TestAccVPCManagedPrefixList_basic (212.50s)
--- PASS: TestAccVPCManagedPrefixList_Entry_cidr (214.65s)
--- PASS: TestAccVPCManagedPrefixList_tags (229.16s)
--- PASS: TestAccVPCManagedPrefixList_updateEntryAndMaxEntry (231.66s)

**Pull request merged**